request-an-access-token.md

```diff
@@ -42,11 +42,11 @@ replaced. This should be no longer than 1 year.
 The "pull request" is the PR that initially requested the token, or requested
 permission scope changes. The PR should describe the permission scopes requested.
 
-Repo                              | Secret name                   | Expiration date | Pull Request                               |
----                               | ---                           | ---             | ---                                        |
-[`nodejs/import-in-the-middle`][] | `RELEASE_PLEASE_GITHUB_TOKEN` | 2025-07-23      | <https://github.com/nodejs/admin/pull/902> |
-[`nodejs/node-core-utils`][]      | `RELEASE_PLEASE_GITHUB_TOKEN` | 2025-09-01      | <https://github.com/nodejs/admin/pull/915> |
-
+Repo                              | Secret name                    | Expiration date | Pull Request                               |
+---                               | ---                            | ---             | ---                                        |
+[`nodejs/import-in-the-middle`][] | `RELEASE_PLEASE_GITHUB_TOKEN`  | 2025-07-23      | <https://github.com/nodejs/admin/pull/902> |
+[`nodejs/node-core-utils`][]      | `RELEASE_PLEASE_GITHUB_TOKEN`  | 2025-09-01      | <https://github.com/nodejs/admin/pull/915> |
+[`nodejs/wasm-builder`][]         | `RELEASE_PLEASE_GITHUB_ACTION` | 2025-10-01      | <https://github.com/nodejs/admin/pull/926> |
```

Comment:
The reference ``[`nodejs/wasm-builder`]`` isn't defined at the bottom of the document

Reply:
@nschonni thanks for catching that added, although the link will not work until the repo is transferred as discussed in https://github.com/nodejs/admin/issues/927

Reply:
No problem. 
PS, the diff would be smaller if this repo used the same ending of `TOKEN` instead of `ACTION`

Reply:
True, but that is a bigger change than just chaging in this repo. As I move over the repo I'll consider doing that.

Reply:
Updated